abi3audit/_cli.py

```diff
@@ -175,55 +175,72 @@ def main() -> None:
         prog="abi3audit",
         description="Scans Python extensions for abi3 violations and inconsistencies",
     )
-    parser.add_argument(
+    version_group = parser.add_argument_group()
+    version_group.add_argument(
+        "-V",
+        "--version",
+        action="store_true",
+        help="show abi3audit version information",
+    )
+    main_group = parser.add_argument_group("main arguments")
+    main_group.add_argument(
```

Comment:
I think this makes the parser a lot harder to follow -- could we use `action='version'` instead? That should handle everything automatically and avoid the need for separate arg groups.

For example:

```python
parser.add_argument("-V", "--version", action='version', version=f"%(prog)s {__version__}")
```

Reply:
Oh that's much better, I didn't know that argparse had built in support for version display.